### ISSUE TITLE
To repeat your result

### ISSUE THREAD

**Issue by kelvinqin** [gheyret/uyghur-asr-ctc#1]
Gheyret, 
Thanks for your work, I tried to reproduce your work, but after 300 epochs (batch size = 20),  CER was still around 10%, not sure how many epochs for you to reach 7.21%
One more thing I noticed that your dataset is a little different with the original dataset (openslr) --- for some utterances, for example, F178_007.wav, you have F178_007_1.wav and F178_007_2.wav. Can you elaborate it a little on the reason for you to do that modification?
Thanks for your answer!

**Comment by gheyret** [comment]
Hi @kelvinqin,

1. I maked 100 hours audio dataset and using it to train. 
	like https://github.com/gheyret/UQSpeechDataset. (It is for TTS, but I use it for ASR too)
2. I split some long audio clip form original data set to shorter audio clip. like F178_007.wav -> F178_007_1.wav and F178_007_2.wav. Because, my GPU have only 11GB memory.  Long audio clips cause memory error.

**Comment by kelvinqin** [comment]
Hi Gheyret,
Thanks for your explanation on why you split some of the utterances. You answer is very clear and understood. 

I just return to my laptop from a whole-day meeting, I found my training task has arrived at the 410th epoch, but the CER is still around 10%. Do you have any suggestion for me to reproduce your result (CER is around 7%)?

**VALIDATION: [CER:10.24% (18363/179264 letters) WER:40.00% (9113/22782 words), Avg loss:0.460283]: 100%**

BTW, one more question, do you have any tool to convert uyghur arahic text into your uyghur latin text? I want to include more data into the training but whose script is in uyghur arabic format.

Thanks,
Kelvin

**Comment by gheyret** [comment]
Hi @kelvinqin,

Thank you for interesting Uyghur.

There many Applications and Web sites to convert Uyghur Arabic Scripts(in Uyghur UEY) to Uyghur Latin Scripts(in Uyghur ULY).
Application: 
   https://github.com/gheyret/UyghurEditPP/releases (Windows application)
Web Site: 
  https://www.yulghun.com/tools/convert.html
  https://uyghuralphabetconverter.herokuapp.com/

**Comment by kelvinqin** [comment]
Hi Gheyret,
Thanks for all the information,

With the same data and training script, after 496 epochs, I got CER = 9.9，can you let me know how you achieve CER = 7.21 please？I am wondering whether I made any mistake.

Many thanks!
Kelvin

**Comment by h310558606** [comment]
Hi kelvin，
I am very interested in Uyghur ASR , can you tell me what is your current CER？
Do you have a better model for Uyghur ASR now?
